docs: add note about prefixed kafka SSL configs (#8910)
confluentinc/ksql: docs/operate-and-deploy/installation/server-config/security.md

refer to the
[Security Guide](https://docs.confluent.io/current/security/index.html).

> You can also prefix the SSL truststore configs with `ksql.streams.` in order

Reviewer: ```suggestion
To use separate trust stores for encrypted communication with {{ site.ak }} and external communication with ksqlDB clients, prefix the SSL truststore configs with `ksql.streams.`:
```

> to use separate trust stores for encrypted communication with Kafka and

Reviewer: ```suggestion
```

> external communication with ksqlDB clients:

Reviewer: ```suggestion
```
